### ISSUE TITLE
Widget support

### ISSUE THREAD

**Issue by rocksdanister** [rocksdanister/lively#161]
**Is your feature request related to a problem? Please describe.**
Turn webpages into desktop widgets; no custom language, turn existing web code into desktop widgets with minimal effort.

**Describe the solution you'd like**
This will add more layer of customization - clocks, [system information](https://github.com/rocksdanister/lively/wiki/Web-Guide-V-:-System-Data), launchers etc..

**Describe alternatives you've considered**
Using Rainmeter software.

**Additional context**
So the idea will be using webview2/cerfsharp for the window with a user defined color of the webpage (say green) turned to transparent (not sure if this is possible yet.)

**Comment by EngrishGuysTaig** [comment]
It mean Rainmeter compatible
?

**Comment by rocksdanister** [comment]
No, Rainmeter is a different software I am not affiliated with.

**Comment by naeem76** [comment]
@rocksdanister The other issue I opened is slightly related to this, but I was initially planning to only support html as full wallpaper, and then in the future to allow multiple html widgets on any kind of wallpaper, in a way similar to rainmeter, but easier to style

**Comment by LilRaven8** [comment]
I can't open rainmeter it's blocked by my admin; I'm using a school comuter ( I'm in 7th grade)
